java/src/com/ibm/streamsx/topology/generator/spl/OperatorGenerator.java

```diff
@@ -89,6 +90,25 @@ private static void portTypesNote(JSONObject op, StringBuilder sb) {
         }
     }
 
+    private void viewAnnotation(JSONObject op, StringBuilder sb){
+	JSONObject config = (JSONObject)op.get("config");
+	if(config == null)
+	    return;
+	JSONObject viewConfig = (JSONObject)config.get("viewConfig");
+	if(viewConfig == null)
+	    return;
+	String name = (String)viewConfig.get("name");
+	String port = (String)viewConfig.get("port");
+	Double bufferTime = (Double)viewConfig.get("bufferTime");
```

Comment:
Can you assume this is a Double? What if the app developer passes in 4?
Probably best to get it as a Number and then call doubleValue() on it.

Similar for sampleSize